lib/strategy/EnrichClientFromRemoteConfigStrategy.js

```diff
@@ -44,35 +47,53 @@ EnrichClientFromRemoteConfigStrategy.prototype._resolveApplicationByName = funct
   });
 };
 
-EnrichClientFromRemoteConfigStrategy.prototype.process = function (config, callback) {
-  var application = config.application;
+// If there are only two applications and one of them is
```

Comment:
The 'Stormpath' application is a reserved application. It will always be created and cannot be deleted.